charts/4allportal/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "3.10.37"
 description: A Helm chart for 4ALLPORTAL version 3.10.0 and up
 name: 4allportal
-version: 19.0.45
+version: 19.0.46
```

Comment:
It is a new feature. We can increment to `19.1.0`.

Reply:
Done